core/src/main/java/inetsoft/storage/LoadKeyValueTask.java

```diff
@@ -80,7 +80,10 @@ public void run() {
             }
 
             map.clear();
-            map.putAll(temp);
+
+            //explicitly pass as TreeMap to prevent extraneous warning that putAll() could cause deadlock
+            Map<String, T> tempAsTree = new TreeMap<>(temp);
```

Comment:
`temp` _is_ a TreeMap. See line 62. Just change the declaration to TreeMap instead of Map.